kiwi/kiwi.py

```diff
@@ -29,7 +29,17 @@ def extras(help_, version, options, doc):
     """
     Overwritten method from docopt
 
-    We want to show our own usage for -h|--help
+    Shows our own usage message for -h|--help
+
+    :param bool help_: indicate to show help
```

Comment:
Why is there a "_" after `help`? Looks a bit strange to me. :wink:

Reply:
because the extras mehod was overwritten by kiwi from docopt (as I wrote in the docstring :))
the docopt original version of the extras methd declared the argument '_help' for whatever reason. yes it's strange but I wanted to keep the same definition